### PR TITLE
Update Quay access statement

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

File: docs/helm-deployment-aws_kops.md [hunk 1]
```diff
@@ -293,7 +293,7 @@ export EFS_SERVER=<EFS_ID>.efs.<AWS-REGION>.amazonaws.com
 ```
 ### Creating a Docker registry pull secret
 
-Since we need to use protected Docker images from Quay.io, you need access to a secret with credentials to be able to pull those images.
+Since we need to use private (Enterprise-only) Docker images from Quay.io, you need access to a secret with credentials to be able to pull those images.
```
Comment (killerboot) [inline docs/helm-deployment-aws_kops.md]: I would rephrase this sentence in the following way:
Since we need to use private (Enterprise-only) Docker images, you need credentials to be able to pull those images from quay.io.

File: README.md [hunk 2]
```diff
@@ -51,4 +51,4 @@ Another typical change would be the integration of your company-wide monitoring
 * [Tips and tricks](https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/docs/tips-and-tricks.md) for working with Kubernetes and Alfresco Content Services.
 * The images downloaded directly from Docker Hub, or Quay.io are for a limited trial of the Enterprise version of Alfresco Content Services that goes into read-only mode after 2 days. Request an extended 30-day trial at
  https://www.alfresco.com/platform/content-services-ecm/trial/docker
-* You'll need your [Quay.io](https://quay.io/) account credentials to access the Docker images. If you don't already have these credentials, contact [Alfresco Support](https://support.alfresco.com/).
+* Alfresco customers can request Quay.io credentials by logging a ticket with [Alfresco Support](https://support.alfresco.com/). These credentials are required to download private (Enterprise-only) Docker images from Quay.io.
```
Comment (killerboot) [inline README.md]: Probably better to use "pull" instead of "download".

File: docs/helm-deployment-aws_kops.md (hunk 1 above)
Comment (anxumalo) [reply docs/helm-deployment-aws_kops.md]: Done

File: README.md (hunk 2 above)
Comment (anxumalo) [reply README.md]: Done

**Comment by anxumalo** [comment]
Thanks for the review
